com.unity.ml-agents.extensions/Runtime/Input/InputActuatorComponent.cs

```diff
@@ -0,0 +1,359 @@
+#if MLA_INPUT_SYSTEM && UNITY_2019_4_OR_NEWER
+using System;
+using System.Collections.Generic;
+using Unity.MLAgents.Actuators;
+using Unity.MLAgents.Policies;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Controls;
+using UnityEngine.InputSystem.Layouts;
+
+namespace Unity.MLAgents.Extensions.Input
+{
+    /// <summary>
+    /// Component class that handles the parsing of the <see cref="InputActionAsset"/> and translates that into
+    /// <see cref="InputActionActuator"/>s.
+    /// </summary>
+    public class InputActuatorComponent : ActuatorComponent
+    {
+        InputActionAsset m_InputAsset;
+        PlayerInput m_PlayerInput;
+        BehaviorParameters m_BehaviorParameters;
+        IActuator[] m_Actuators;
+        InputDevice m_Device;
+        Agent m_Agent;
+        uint m_LocalId;
+
+        static uint s_DeviceId;
```

Comment:
Can you add some comments on what this is used for?

Reply:
yes, I also don't like this very much.